src/Fixer/LanguageConstruct/DirConstantFixer.php

```diff
@@ -44,7 +44,7 @@ public function getDefinition(): FixerDefinitionInterface
      */
     public function isCandidate(Tokens $tokens): bool
     {
-        return $tokens->isTokenKindFound(T_FILE);
+        return $tokens->isAnyTokenKindsFound([T_STRING, T_FILE]);
```

Comment:
if there is T_STRING but no T_FILE, why to run this fixer?

Reply:
should be `all kinds found (string, file)` indeed